pages.de/common/base64.md

```diff
@@ -0,0 +1,20 @@
+# base64
+
+> Kodierung oder Dekodierung von Dateien oder Standardeingaben in und aus Base64, zur Standardausgabe (stdout).
```

Comment:
```suggestion
> Kodierung oder Dekodierung von Dateien oder Standardeingaben in/aus Base64, zur Standardausgabe.
```
Would this be better? What do you think?

> Note: I am not a German speaker. I used Google translate to validate this translation. My suggestion is based on the English page.

Reply:
Yes this seems better